app/services/html/parser.rb

```diff
@@ -42,6 +43,12 @@ def prefix_all_images(width = 880)
         # allow image to render as-is
         next if allowed_image_host?(src)
 
+        if synchronous_detail_detection
+          height_width = image_height_width(img)
+          img["width"] = height_width[0]
+          img["height"] = height_width[1]
```

Comment:
I think this would read a bit better:

```suggestion
          width, height = image_height_width(img)
          img["width"] = width
          img["height"] = height
```